igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitorSpec.groovy

```diff
@@ -212,6 +212,10 @@ class TravisBuildMonitorSpec extends Specification {
                 it.content.project.lastBuild.number == 4
         })
 
+        1 * travisBuildMonitor.echoService.postEvent({
```

Comment:
Added this assert to reflect the current behavior. Any objections?

Reply:
Great:D